src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -144,13 +144,13 @@ export const monoPlatform: Platform = {
     return ((baseAddress as any as number) + (fieldOffset || 0)) as any as T;
   },
 
-  beginHeapLock: function() {
+  beginHeapLock: function () {
```

Comment:
Was this intentional? `.eslintrc.js` is configured to treat this as a styling error.